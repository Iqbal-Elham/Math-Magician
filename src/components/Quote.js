import React from 'react';
import styles from './styles/Quote.module.css';

const Quote = () => (
  <div className={styles.content}>
    <p className={styles.quote}>
      Mathematics is not about numbers, equations, computations, or algorithmic,
      it is about understanding. - William Paul Thurston
    </p>
  </div>
);

export default Quote;
