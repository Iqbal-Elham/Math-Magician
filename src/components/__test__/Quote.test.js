import renderer from  'react-test-renderer';
import Quote from '../Quote';
â€‹
it("Render the Quote user interaction", () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
})