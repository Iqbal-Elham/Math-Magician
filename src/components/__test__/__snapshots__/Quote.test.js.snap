// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of the Quote page 1`] = `
<div
  className="content"
>
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithmic, it is about understanding. - William Paul Thurston
  </p>
</div>
`;
